using-timescaledb/data-retention.md

```diff
@@ -20,6 +27,42 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention with Continuous Aggregates
+
+The function `drop_chunks` can be used to control data retention for
+the continuous aggregate as well, but if you have a continuous
+aggregate defined on the table you need to provide a value for the
+`cascade_to_materialization` parameter. Not giving a value when there is a
+continuous aggregate defined on a hypertable will generate an error.
+
+Providing the value `FALSE` to `cascade_to_materialization` will only
+remove chunks from the hypertable and not cascade to dropping chunks
+on the continuous aggregate.
+
+It is not possible to remove chunks from the hypertable without
```

Comment:
```suggestion
It is possible to remove chunks from the hypertable after removing
```
use positive instead of double negative. I think that is easier to parse.

Reply:
Done.